server/api/models.py

```diff
@@ -239,5 +240,91 @@ class Meta:
         unique_together = ('user', 'event')
 
     def __str__(self):
-        return "@{} is attending event {}" .format(
-            self.user.slack_name, self.event.title)
+        return "@{} is attending event {}" \
+            .format(self.user.slack_name, self.event.title)
+
+
+class UserEventHistory(BaseInfo):
+    """User Event History Model definition"""
+
+    CR = 'CREATE'
+    DE = 'DEACTIVATE'
+    UP = 'UPDATE'
+    AT = 'ATTEND'
+    RE = 'REJECT'
+
+    USER_EVENT_ACTIONS = ((CR, 'CREATE'), (DE, 'DEACTIVATE'),
+                          (UP, 'UPDATE'), (AT, 'ATTEND'), (RE, 'REJECT'))
+
+    andela_user_profile = models.ForeignKey(AndelaUserProfile,
+                                            on_delete=models.CASCADE)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE)
+    user_event_action = models.CharField(max_length=10, default=CR,
+                                         choices=USER_EVENT_ACTIONS)
+
+    class Meta:
+        ordering = ('-created_at',)
+
+    def __str__(self):
+        return f'{self.andela_user_profile.slack_name} ' \
+               f'{self.user_event_action} ' \
```

Comment:
Why are we using multiple strings literal when breaking into multiple lines?

Reply:
Thanks @makindemayowa

Reply:
I believe something of this nature should still work
`name = 'Fred'`
`age = 50`
 `print(f'{name} \`
  `man \`
  `{age}')`

Reply:
Something of this nature might work

`man = "yes boss"`
`woman = "shawthy"`
`others = "bobrisjy"`
`f'{man} \`
`{woman} \`
`{others}'`

Reply:
Thanks, @makindemayowa but with this approach the output when will be `'yes boss {woman} {others} '`  instead of `'yes boss shawthy bobbrisjy'`. Please check as well for confirmation.

Reply:
It sure works.
<img width="796" alt="screen shot 2018-08-03 at 12 13 30 pm" src="https://user-images.githubusercontent.com/25981851/43640150-add9778c-9716-11e8-9034-5d69cc840506.png">

Reply:
Thanks, @makindemayowa My bad... Will fix that now. Thanks a lot.